buildman/manager/executor.py

```diff
@@ -402,6 +406,19 @@ def __init__(self, *args, **kwargs):
     def running_builders_count(self):
         q = {"labelSelector": "build,time,manager,quay-sha"}
         jobs_list = self._request("GET", self._jobs_path(), params=q)
+        if jobs_list.status_code != 200:
```

Comment:
(nitpick) It might more useful to future developers if this was renamed from `jobs_list` to `req` and then use the variable name `jobs_list` to reference the actual list of jobs.